p4rt_app/p4rt.cc

```diff
@@ -325,8 +313,9 @@ int main(int argc, char** argv) {
       std::make_unique<p4rt_app::sonic::SystemCallAdapter>(),
       p4rt_app::sonic::PacketIoOptions{});
 
-  // Wait for PortInitDone to be done.
-  p4rt_app::WaitForPortInitDone();
+  // Create a netdev translator for P4Runtime's PacketIo handling.
+// TODO
+//  swss::IntfTranslator netdev_translator(&packetio_config_db);
```

Comment:
Can you pls follow this convention for adding a TODO. For multi-line code, do not add the code under the comment.

// TODO(PINS): Create a netdev translator for P4Runtime's PacketIo handling.
// swss::IntfTranslator netdev_translator(&packetio_config_db);

Reply:
@kishanps, I changed the code to have TODO at the beginning of the comment.